### DISCUSSION TITLE
What does TestRunProfileKind.Coverage do?

### DISCUSSION THREAD

**Discussion by rwalle** [microsoft/vscode-discussions#982] [Extension Development QnA] [answered]
I tried to register a test profile with `kind = TestRunProfileKind.Coverage`, i.e.
```js
controller.createRunProfile('Run Coverage', TestRunProfileKind.Coverage, () => {});
```

However I cannot see the new profile anywhere -- it is not in the test explorer or the context menu of the gutter. I don't have such issues with "Run" or "Debug" values, the profiles show up as intended. Does anyone has experience with this?

Thanks

References:
* [TestController](https://code.visualstudio.com/api/references/vscode-api#TestController)
* [TestRunProfileKind](https://code.visualstudio.com/api/references/vscode-api#TestRunProfileKind)

**Comment by r3m0t** [comment, marked as answer]
Hi, test coverage support is coming early next year. Check the issue tracker.
